src/Files.App/Data/Models/ColumnsViewModel.cs

```diff
@@ -12,14 +12,55 @@ public class ColumnsViewModel : ObservableObject
 			UserLength = new GridLength(24, GridUnitType.Pixel),
 			IsResizeable = false,
 		};
-
 		[LiteDB.BsonIgnore]
 		public ColumnViewModel IconColumn
 		{
 			get => iconColumn;
 			set => SetProperty(ref iconColumn, value);
 		}
 
+		private ColumnViewModel _GitStatusColumn = new();
+		public ColumnViewModel GitStatusColumn
+		{
+			get => _GitStatusColumn;
+			set => SetProperty(ref _GitStatusColumn, value);
+		}
+
+		private ColumnViewModel _GitLastCommitDateColumn = new();
+		public ColumnViewModel GitLastCommitDateColumn
+		{
+			get => _GitLastCommitDateColumn;
+			set => SetProperty(ref _GitLastCommitDateColumn, value);
+		}
+
+		private ColumnViewModel _GitLastCommitMessageColumn = new();
+		public ColumnViewModel GitLastCommitMessageColumn
+		{
+			get => _GitLastCommitMessageColumn;
+			set => SetProperty(ref _GitLastCommitMessageColumn, value);
+		}
+
+		private ColumnViewModel _GitCommitAuthorColumn = new();
+		public ColumnViewModel GitCommitAuthorColumn
+		{
+			get => _GitCommitAuthorColumn;
+			set => SetProperty(ref _GitCommitAuthorColumn, value);
+		}
+
+		private ColumnViewModel _GitLastCommitShaColumn = new();
+		public ColumnViewModel GitLastCommitShaColumn
+		{
+			get => _GitLastCommitShaColumn;
+			set => SetProperty(ref _GitLastCommitShaColumn, value);
+		}
+
+		public bool IsAllGitColumnsHidden =>
```

Comment:
What's this bool used for?

Reply:
Collapsing git columns menuflyoutsubitem.

Reply:
Can we use the IsGitRepo property instead?

Reply:
Sure.

Reply:
Can this be removed then?

Reply:
Removed

Reply:
Thank you